### PR TITLE
docs: fix various typos and small things

### PR DESCRIPTION
**Pull request by fredericmoulins**
Applied on buster-dev. Most things apply to the dev branch also, but not everything.
Let me know if you want to apply to dev first.

### CODE REVIEW

File: docs/backup-home.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ backup:
 
 ## Remote server over SSH
 
-This location scheme is using borg on a remote server, with an SSH connection. Therefore, the borg software need to be
+This location scheme is using borg on a remote server, with a SSH connection. Therefore, the borg software needs to be
```
Comment (arodier) [inline docs/backup-home.md:91]: Nope, sorry. We say an SSH, not a SSH ;-) The rule is how to pronounce the next word, not the console.

Comment (arodier) [reply docs/backup-home.md:91]: All the rest is good! Thanks a lot for the amending, very useful to developers and users too.

Comment (arodier) [reply docs/backup-home.md:91]: This is fine, I merge and will amend this ;-)

Comment (fredericmoulins) [reply docs/backup-home.md:91]: > Nope, sorry. We say an SSH, not a SSH ;-) The rule is how to pronounce the next word, not the console.

Ha, the sound… ok, noted, thanks :)

**Pull request merged**